lib/potluck.rb

```diff
@@ -0,0 +1,57 @@
+require './lib/dish'
+require 'pry'
+
+class Potluck
+  attr_accessor :date,
+                :dishes
+  def initialize(date)
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish(new_dish)
+    @dishes << new_dish
+  end
+
+  def get_all_from_category(selected_category)
+    array_to_return = []
+    @dishes.each do |dish|
+      if dish.category == selected_category
+        array_to_return << dish
+      else
+        next
+      end
+    end
+    array_to_return
+  end
+
+  def menu
+    hash_to_return = {}
+    @dishes.each do |dish|
+    if hash_to_return.include?(dish.category) == false
```

Comment:
Consider using `.nil?` to simplify this:

```
hash_to_return[dish.category].nil?
```

Also, recommend indenting this section to indicate it exists within the `each` block that starts on line 30.